libraries/AP_Motors/AP_MotorsMulticopter.cpp

```diff
@@ -798,3 +826,36 @@ void AP_MotorsMulticopter::convert_pwm_min_max_param(int16_t radio_min, int16_t
     _pwm_min.set_and_save(radio_min);
     _pwm_max.set_and_save(radio_max);
 }
+
+bool AP_MotorsMulticopter::pre_arm_check()
+{
+#if HAL_WITH_ESC_TELEM
+    if (is_zero(_fs_rpm)) {
```

Comment:
Maybe:
```suggestion
    if (!is_positive(_fs_rpm)) {
```
So -1 also works for disabled

Reply:
I prefer 0 for disabled if possible..

Reply:
Yes, 0 for disable, but negative values are not a valid RPM.